examples/2_Intermediate/boozerQA_ls_mpi.py

```diff
@@ -77,6 +77,7 @@
 boozer_surfaces = boozer_surfaces[:nsurfaces]
 ress = ress[:nsurfaces]
 for boozer_surface, res in zip(boozer_surfaces, ress):
+    boozer_surface.options['solver'] = ['ls', 'ls_newton']
     boozer_surface.run_code(res['iota'], res['G'])
 
 mpi_surfaces = MPIOptimizable(surfaces, ["x"], comm)
```

Comment:
can you ensure the same answer is given on this example when run in parallel vs on a single core after these changes?